core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -1108,17 +700,18 @@ public int next() {
 
         DescendingIter() {
             from = null;
-            stack = new IntStack(entries.length / FIELDS);
+            stack = new IntStack(entrySet.getNumOfEntries());
```

Comment:
I really like all these changes that remove references of version, FIELD, length, from all over
good job 
but I cant really verify it is done correctly

Reply:
First, thank you! Second, I multi-reviewed the changes and our testing is pretty good! Third, Liran has a good eye! Finally, God bless Oak, lets hope for good :)